lib/core_ext/em-hiredis.rb

```diff
@@ -0,0 +1,12 @@
+module EventMachine
```

Comment:
I don't believe this is an issue upstream, I think it's caused by the delegators, this is just a workaround for it